components/Pages/Companies/NextDeveloper.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+// My components
+import Button from '../../Button'
+import H2 from '../../H2'
```

Comment:
Ya no deberiamos usar esta forma de importar, 

Agregue al archivo de configuración de TS la configuración del basedir, puedes/debes poner el nombre de la carpeta directamente, asi los componentes ya no dependen de estar especificamente en una carpate, puden ser movidos de carpeta y aun funciona todo

Esto quedaria así 
```suggestion
import Button from 'components/Button'
import H2 from 'components/H2'
```

Lo mismo en los de mas archivos